bingo/symbolic_regression/agraph/string_generation.py

```diff
@@ -41,10 +43,24 @@
                    ABS: "|{}|",
                    SQRT: "\\sqrt{{ {} }}",
                    SAFE_POWER: "(|{}|)^{{ ({}) }}"}
+SYMPY_PRINT_MAP = {ADDITION: "{} + {}",
+                   SUBTRACTION: "{} - ({})",
+                   MULTIPLICATION: "({})*({})",
+                   DIVISION: "({})/({})",
+                   SIN: "sin({})",
+                   COS: "cos({})",
+                   SINH: "sinh({})",
+                   COSH: "cosh({})",
+                   EXPONENTIAL: "exp({})",
+                   LOGARITHM: "log({})",
+                   POWER: "({})**({})",
+                   ABS: "abs({})",
+                   SQRT: "sqrt({})",
+                   SAFE_POWER: "abs({})**({})"}
 CONSOLE_PRINT_MAP = {ADDITION: "{} + {}",
                      SUBTRACTION: "{} - ({})",
                      MULTIPLICATION: "({})({})",
-                     DIVISION: "({})/({}) ",
+                     DIVISION: "({})/({}) ",  # TODO extra space intentional?
```

Comment:
I was curious if this extra space was intentional

Reply:
not intentional, this is a typo

Reply:
sounds good, removed in new changes